evalml/pipelines/components/estimators/regressors/arima_regressor.py

```diff
@@ -152,7 +152,9 @@ def _format_dates(self, dates, X, y, predict=False):
             forecasting_ = import_or_raise(
                 "sktime.forecasting.base", error_msg=arima_model_msg
             )
-            fh_ = forecasting_.ForecastingHorizon(dates, is_relative=False)
+            fh_ = forecasting_.ForecastingHorizon(
+                [i + 1 for i in range(len(dates))], is_relative=True
```

Comment:
Why are we forecasting on this array of ints rather than the dates? Do the dates become irrelevant when we use `is_relative`?

Reply:
@bchen1116 Correct, the ints represent how many "periods" after relative to the end of the training dates we want to predict for.